src/main/java/net/fabricmc/loom/configuration/providers/mappings/MojangMappingsDependency.java

```diff
@@ -128,12 +128,14 @@ public ExternalModuleDependency setChanging(boolean changing) {
 	@Override
 	public Set<File> resolve() {
 		Path mappingsDir;
+
 		try {
 			mappingsDir = extension.getMappingsProvider().getMappedVersionedDir(String.format("mojang/%s.%s-%s", GROUP, MODULE, getVersion()));
```

Comment:
Isn't this supposed to do the version already?

Reply:
The folder contains the Minecraft version, but it will create a `mappings.tiny-final.jar` file in the ~/.gradle/caches/fabric-loom/
And that is the same for every MC version